osu.Game/Screens/Play/BreakOverlay.cs

```diff
@@ -109,10 +130,36 @@ protected override void LoadComplete()
             initializeBreaks();
         }
 
-        private void initializeBreaks()
+        protected override void Update()
+        {
+            base.Update();
+            updateBreakTimeBindable();
+        }
+
+        private void updateBreakTimeBindable()
         {
-            if (!IsLoaded) return; // we need a clock.
+            if (breaks?.Any() != true)
```

Comment:
Just a note for the future - we likely don't want to use any linq in per-update scenarios:

| Method |      Mean |     Error |    StdDev | Gen 0/1k Op | Gen 1/1k Op | Gen 2/1k Op | Allocated Memory/Op |
|------- |----------:|----------:|----------:|------------:|------------:|------------:|--------------------:|
|  Count |  1.788 ns | 0.0117 ns | 0.0104 ns |           - |           - |           - |                   - |
|    Any | 17.875 ns | 0.1902 ns | 0.1686 ns |      0.0127 |           - |           - |                40 B |